src/main/java/io/github/opencubicchunks/cubicchunks/server/level/progress/CubicChunkProgressListener.java

```diff
@@ -0,0 +1,16 @@
+package io.github.opencubicchunks.cubicchunks.server.level.progress;
+
+import javax.annotation.Nullable;
+
+import io.github.opencubicchunks.cubicchunks.world.level.chunklike.CloPos;
+import net.minecraft.world.level.chunk.ChunkStatus;
+
+public interface CubicChunkProgressListener {
+    void cc_updateSpawnPos(CloPos pCenter);
+
+    void cc_onStatusChange(CloPos pChunkPosition, @Nullable ChunkStatus pNewStatus);
+
+    void start();
+
+    void stop();
```

Comment:
cc prefix? or is it required to not have one? perhaps a comment

Reply:
these methods are actually unnecessary; I have removed them (I coped the entire ChunkProgressListener interface originally)